stock_operating_unit/demo/stock_demo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 Eficent Business and IT Consulting Services S.L.
+     Copyright 2016-2017 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo noupdate="1">
+
+    <!-- Shelf 2 stock location -->
+    <record id="stock.stock_location_14" model="stock.location">
+        <field name="operating_unit_id" ref="operating_unit.main_operating_unit"/>
+    </record>
+
+    <!-- Shelf 1 stock location -->
+    <record id="stock.stock_location_components" model="stock.location">
+        <field name="operating_unit_id" ref="operating_unit.main_operating_unit"/>
+    </record>
+
+    <!-- Multi Company -->
+    <!-- Child Company 1-->
+    <record id="stock.res_company_1" model="res.company">
+        <field name="name">Your company child</field>
+    </record>
+
+    <record id="operating_unit_shop0" model="operating.unit">
+        <field name="name">Chicago</field>
+        <field name="code">CH</field>
+        <field name="partner_id" ref="stock.res_partner_company_1"/>
+        <field name="company_id" ref="stock.res_company_1"/>
+    </record>
+
+    <!-- Chicago Warehouse -->
+    <record id="stock.stock_warehouse_shop0" model="stock.warehouse">
+        <field name="operating_unit_id" ref="operating_unit_shop0"/>
+    </record>
+
+    <!-- Your Company, Chicago shop -->
+    <record id="stock.stock_location_shop0" model="stock.location">
+        <field name="operating_unit_id" ref="operating_unit_shop0"/>
```

Comment:
The problems seems to be a <code>stock.warehouse</code> called 'My Company, Chicago' that is only created as demo data in v10. I didn't find where that record is created though, so don't know what is the external_id to add to it the correct  operating_unit_id when installing this module. I think that would solve the problem.